### PR TITLE
Add projects and project boards in exposed metrics

### PR DESCRIPTION
**Pull request by romdum**
Today, there is no metrics for projects. 

I added "gitea_projects" and "gitea_projects_boards" in exposed metrics.

### CODE REVIEW

**Review by 6543 — APPROVED** [review]

File: models/statistic.go [hunk 1]
```diff
@@ -62,5 +63,7 @@ func GetStatistic() (stats Statistic) {
 	stats.Counter.HookTask, _ = db.GetEngine(db.DefaultContext).Count(new(HookTask))
 	stats.Counter.Team, _ = db.GetEngine(db.DefaultContext).Count(new(Team))
 	stats.Counter.Attachment, _ = db.GetEngine(db.DefaultContext).Count(new(Attachment))
+	stats.Counter.Project, _ = db.GetEngine(db.DefaultContext).Count(new(Project))
+	stats.Counter.ProjectBoard, _ = db.GetEngine(db.DefaultContext).Count(new(ProjectBoard))
```
Comment (KN4CK3R) [inline models/statistic.go]: While working on it, could you refactor this to
```go
e := db.GetEngine(db.DefaultContext)
stats.Counter.Comment, _ = e.Count(new(Comment))
...
stats.Counter.Project, _ = e.Count(new(Project))
stats.Counter.ProjectBoard, _ = e.Count(new(ProjectBoard))
```

Comment (romdum) [reply models/statistic.go]: Done

**Review by KN4CK3R — APPROVED** [review]

File: modules/metrics/collector.go [hunk 2]
```diff
@@ -119,6 +121,16 @@ func NewCollector() Collector {
 			"Number of Organizations",
 			nil, nil,
 		),
+		Projects: prometheus.NewDesc(
+			namespace+"projects",
+			"Number of project",
```
Comment (delvh) [inline modules/metrics/collector.go]:
```suggestion
			"Number of projects",
```

**Review by delvh — APPROVED** [review]

**Comment by 6543** [comment]
:rocket:

**Pull request merged**